PyLaTeXtables/templates/eoc.tex

```diff
@@ -4,7 +4,7 @@
 
 ((* set tabular_columns = index_column_types * index_columns + 'r' * ( data_columns % 2 ) + 'rc' * ( data_columns // 2 ) *))
 ((* set np_digits = 2 *))
-((* set header_is_hierarchical = df.columns.format(sparsify=True) != df.columns.format(sparsify=False) *))
+((* set header_is_hierarchical = df.columns.nlevels > 1 *)) % to check df.columns is hierarchical MultiIndex
```

Comment:
The `%`-style comment will appear in the rendered output, which does not make much sense. There should be a template-level comment, which is `((# ... #))` (look for `comment_start_string` in the Python code).